### PR TITLE
feat(date-time-condition): Added default date-time condition definition

### PR DESCRIPTION
**Pull request by agkm10**
## **Description**

INSERT A SMALL DESCRIPTION OF WHAT YOU CHANGED HERE

#### **Verify that...**

- [x] Any related demos were added and `npm start` and `npm run build` still works
- [x] New demos work in `Safari`, `Chrome` and `Firefox`
- [x] `npm run lint` passes
- [x] `npm test` passes and code coverage is increased
- [x] `npm run build` still works

#### **Bullhorn Internal Developers**
- [x] Run `Novo Automation`

##### **Screenshots**

### CODE REVIEW

File: projects/novo-elements/src/elements/date-time-picker/DateTimePicker.ts [hunk 1]
```diff
@@ -224,15 +231,15 @@ export class NovoDateTimePickerElement implements ControlValueAccessor {
   writeValue(model: any): void {
     this.model = model;
     if (Helpers.isEmpty(model)) {
-      this.model = new Date();
+      this.model = this.createFullDateValue(this.datePickerValue, this.timePickerValue);
```
Comment (hiqbal01) [inline projects/novo-elements/src/elements/date-time-picker/DateTimePicker.ts:234]: wouldn't this be a change in behavior for existing date time pickers?

File: projects/novo-elements/src/elements/field/formats/date-time-format.ts [hunk 2]
```diff
@@ -160,7 +160,7 @@ export class NovoDateTimeFormatDirective extends IMaskDirective<any> implements
     const dateTime =  text.split(', ');
     const hour: string = dateTime[1].slice(0, 2);
     if (!this.military) {
-      const input = dateTime[1].substr(17, 4).replace(/\-/g, '').trim().slice(0, 2);
+      const input = dateTime[1].substr(5, 4).replace(/\-/g, '').trim().slice(0, 2);
```
Comment (hiqbal01) [inline projects/novo-elements/src/elements/field/formats/date-time-format.ts:163]: what does this change represent? was this previously broken?

Comment (agkm10) [reply projects/novo-elements/src/elements/field/formats/date-time-format.ts:163]: This was a bug that was found from when I made the date-time formatter. I was working off the full string not the split string.

File: projects/novo-elements/src/elements/query-builder/query-builder.module.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import { NovoDateTimePickerModule } from './../date-time-picker/DateTimePicker.module';
```
[the remark this answers is not in the document]
Comment (agkm10) [reply projects/novo-elements/src/elements/query-builder/query-builder.module.ts:1]: Will fix

File: projects/novo-elements/src/elements/date-time-picker/DateTimePicker.ts (hunk 1 above)
Comment (agkm10) [reply projects/novo-elements/src/elements/date-time-picker/DateTimePicker.ts]: Changed it back. I had updated it to fix something but it did not change the behavior.

**Review by dvoegelin — APPROVED** [review]

**Pull request merged**